ptbr/tests/test_validation.py

```
@@ -0,0 +1,381 @@
+
+
+# ===================================================================
+# 6. CLI smoke test
+# ===================================================================
+
+def test_cli_validate():
+    sample = Path(__file__).resolve().parents[2] / "examples" / "sample_data.json"
+    if not sample.exists():
+        report("CLI validate", False, "sample_data.json not found")
+        return
+    ret = os.system(f'python -m ptbr --file-or-repo "{sample}" --validate > /dev/null 2>&1')
```

Ruff (0.15.1)

[error] Starting a process with a shell, possible injection detected

(S605)